kivy/input/providers/mouse.py

```diff
@@ -127,6 +127,15 @@ def clear_graphics(self, win):
             win.canvas.after.remove(de[1])
 
 
+class MouseHoverEvent(HoverEvent):
```

Comment:
Documentation missing too

Reply:
Will be added that the end of development.